src/de/jost_net/JVerein/server/EinstellungImpl.java

```diff
@@ -743,6 +743,22 @@ public void setRechnungTextBar(String rechnungtextbar) throws RemoteException
   {
     setAttribute("rechnungtextbar", rechnungtextbar);
   }
+  
+  @Override
+  public int getZaehlerLaenge() throws RemoteException 
+  {
+    try {
+      return (int) getAttribute("zaehlerlaenge");
```

Comment:
Hier würde ich noch auf ungültige Werte < 1 testen und in dem Fall ebenfalls 5 zurückliefern.

Reply:
Das Inputfeld Zählerlänge lässt nur natürliche Zahlen mit 0 zu. Deshalb sehe ich von der Prüfung < 0 ab. 0 ist auch ein gültiger Wert. Der Zähler wird trotzdem angezeigt.